### PR TITLE
Adding .gitignore

### LINKED ISSUE #7: Add .gitignore
Currently, after installation, all packages installed with the Composer appearing in git as newly added files (see screenshot). A good practice is to add the whole `vendor` folder to git exclusions as they don't belong to the project files.

Local `config.php` should be excluded as well because of 2 main reasons:
- sensitive credentials must not be stored in the version control system
- credentials differ for different developers and this will cause constant merge conflicts

Screenshot 2019-11-03 at 12 05 37

### PR DESCRIPTION
**Pull request by MagedAhmad**
Fixes #7

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/node_modules
```
Comment (vladimino) [inline .gitignore:1]: This line doesn't belong to this change, right? :)

Comment (MagedAhmad) [reply .gitignore:1]: @vladimino Well i thought it belongs here , as im adding folders and files that i suppose to ignore so  i figured that i have to ignore this folder.

Comment (vladimino) [reply .gitignore:1]: For simplicity, I'll accept this. But please re-read my e-mail about collaboration ;)

**Review by vladimino — APPROVED** [review]
LGTM
